setup.py

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env python
-
-from distutils.core import setup
+from setuptools import setup, find_packages
 
 setup(
     name='rtmbot',
-    version='0.2.0',
+    version='0.2.11',
```

Comment:
Good call, I was messing with this some for testing and forgot to update it.

I wonder if we should bump this to 1.0 soon, now that it's going in PyPi and our list of bugs/etc is fairly low.